query-translator/tests/translate.rs

```diff
@@ -672,12 +672,15 @@ fn test_compound_with_ground() {
 
 #[test]
 fn test_unbound_attribute_with_ground() {
-    // TODO: this needs to expand the type code. #475.
-    let query = r#"[:find ?x :where [?x _ ?v] (not [(ground 5) ?v])]"#;
+    let query = r#"[:find ?x ?v :where [?x _ ?v] (not [(ground 17) ?v])]"#;
```

Comment:
Can you assert this does the right thing if it's `?e` in the entity place, so that the ground is interpreted as a ref?  Or verify that it _doesn't_ do the right thing (the ambiguous choice we made earlier, perhaps)?